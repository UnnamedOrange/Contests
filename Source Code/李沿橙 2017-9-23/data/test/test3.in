5
6
1 3 4 5 7 10 
50
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 981 982 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998 999 1000 
18
68 15 25 35 69 36 1 2 3 60 4 5 48 58 67 37 38 59 
4
1 1 8 7 
50
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1000 
