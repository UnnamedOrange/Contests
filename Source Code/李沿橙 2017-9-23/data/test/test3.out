1 3 4 7 10 5 
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 981 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998 999 1000 982 30 
1 2 3 4 5 25 36 37 38 58 59 60 67 69 68 48 35 15 
1 1 7 8 
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1000 
